catalog/ui/catalog-ui-search/src/main/webapp/js/controllers/cesium.layerCollection.controller.js

```diff
@@ -45,6 +46,13 @@ define(['underscore',
             this.collection.forEach(function (model) {
                 var type = imageryProviderTypes[model.get('type')];
                 var initObj = _.omit(model.attributes, 'type', 'label', 'index', 'modelCid');
+
+                /* Set the tiling scheme for WMTS imagery providers that are EPSG:4326 */
+                var projection = properties.projection;
+                if(model.get('type') === "WMT" && projection === "EPSG:4326") {
```

Comment:
Since the projection variable is only used here, let's reference properties.projection directly.